.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'type: enhancement'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I'm not sure how to interpret "problem" here. Did you mean something like "A situation you're trying to understand?"

Reply:
With the example I read it as "Is your feature request related to wanting a feature?"